src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -1603,6 +1607,29 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp32_activation_eltwise_diff_sizes, :
     conv_eltw_test_params{ CASE_CONV_ELTW_FP32_8, 3, 3, 4 },
 }));
 
+class conv_fp16_activation_eltwise_diff_sizes : public ConvEltwTest {};
+TEST_P(conv_fp16_activation_eltwise_diff_sizes, basic) {
+    auto p = GetParam();
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("weights", get_mem(get_weights_layout(p))),
+        data("bias", get_mem(get_bias_layout(p))),
+        data("eltwise_data", get_mem(layout{ p.data_type, p.input_format, p.eltw_shape })),
+        convolution("conv_prim", input_info("input"), { "weights" }, { "bias" }, p.groups, p.stride, p.pad, p.dilation),
+        activation("activation", input_info("conv_prim"), activation_func::relu_negative_slope),
+        eltwise("sum", { input_info("activation"), input_info("eltwise_data") }, eltwise_mode::sum, data_types::f16),
+        reorder("reorder_bfyx", input_info("sum"), p.default_format, data_types::f16)
+    );
+
+    tolerance = default_tolerance(p.default_type);
+    execute(p);
+}
+
+INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_fp16_activation_eltwise_diff_sizes, ::testing::ValuesIn(std::vector<conv_eltw_test_params>{
+    conv_eltw_test_params{ CASE_CONV_ELTW_FP16_SWAP_XY_1, 3, 3, 4 },
+    conv_eltw_test_params{ CASE_CONV_ELTW_FP16_SWAP_XY_2, 3, 3, 4 },
```

Comment:
It is about fusing? I think it is just about convolution..

Reply:
It will test both 1d conv and fusing.

Reply:
I think it will not report any issues related to xy swap itself as ref network may be impacted by same issue. So please add separate test case for convolution primitive only

Reply:
Add another unit test.